src/Compiler/Microsoft.NET.Sdk.Razor.SourceGenerators/IncrementalValueProviderExtensions.cs

```diff
@@ -6,6 +6,7 @@
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Razor;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
```

Comment:
Looks unused.

Reply:
It's used for the `AnalyzerConfigOptionsProvider`